packages/devtools_app/lib/src/shared/screen.dart

```diff
@@ -76,6 +76,8 @@ enum ScreenMetaData {
     iconAsset: 'icons/app_bar/network.png',
     requiresDartVm: true,
     tutorialVideoTimestamp: '?t=547',
+    // ignore: avoid_redundant_argument_values, false positive
+    worksWithOfflineData: FeatureFlags.networkOffline,
```

Comment:
please also set the value of `requiresConnection`:
```
// ignore: avoid_redundant_argument_values, false positive
requiresConnection: FeatureFlags.networkOffline,
```

Reply:
done